man/zram-generator.conf.md

```diff
@@ -104,6 +104,12 @@ Devices with the final size of *0* will be discarded.
 
   Also see systemd-makefs(8).
 
+* `mount-options` =
+
+  Specifies how the device shall be mount with options. The default is *discard* for enable TRIM support for ext4.
```

Comment:
Please drop the part about chaning the default options for now.

Reply:
Done